ironfish/src/fileStores/config.ts

```diff
@@ -238,9 +237,86 @@ export type ConfigOptions = {
   explorerTransactionsUrl: string
 }
 
+// Matches either an empty string, or a string that has no leading or trailing whitespace.
+const reNoWhitespaceBegEnd = /^[^\s]+(\s+[^\s]+)*$|^$/
+
+// config validation error messages
+const cfgErrMsgPathWhiteSpace = 'Path should not contain leading or trailing whitespace.'
+const cfgErrMsgURL = 'Invalid URL.'
```

Comment:
nit - since these strings are only used once each in `noWhitespaceBegEnd` and `isUrl`, below, I think we could use them in place there without defining separate variables

Reply:
Good call.
I had it this way before I added `isUrl` and `noWhitespaceBegEnd` , so was using the error messages multiple times in the config schema directly.

Reply:
Addressed in https://github.com/iron-fish/ironfish/pull/2359/commits/4f4a7c0136835c516b5bdf3826a4669b426f88bd.